src/02-data-types.js

```diff
@@ -79,8 +89,28 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  if (text.includes(character)) {
+    return true
+  }
+  else {
+    return false
+  }
+}
```

Comment:
Same point as with `compareTypes`. `.includes` already gives you the value you want to return, no need to check its value before returning it.